server/src/test/java/org/opensearch/index/store/remote/filecache/FileCacheTests.java

```diff
@@ -65,6 +71,17 @@ private void createFile(String indexName, String shardId, String fileName) throw
         Files.write(filePath, "test-data".getBytes());
     }
 
+    public void testCreateCacheWithSmallSegments() {
```

Comment:
This test was removed with a recent change. Please remove the above mentioned conditional check in `FileCacheFactory` and this test.